scale/job/management/commands/scale_post_steps.py

```diff
@@ -71,6 +76,10 @@ def handle(self, **options):
                 exit_code = NFS_EXIT_CODE
             elif isinstance(ex, IOError):
                 exit_code = IO_EXIT_CODE
+            elif isinstance(ex, InvalidResultsManifest):
```

Comment:
The invalid results manifest exception should also be thrown when file paths within the results manifest are invalid, not just when the JSON schema itself is invalid.